src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/entity/PersonAddressEntity.kt

```diff
@@ -0,0 +1,35 @@
+package uk.gov.justice.digital.hmpps.personrecord.jpa.entity
+
+import jakarta.persistence.CascadeType
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.Table
+import jakarta.persistence.Version
+
+@Entity
+@Table(name = "person_address")
+class PersonAddressEntity(
```

Comment:
minor but can we just call this AddressEntity? Is there going to be any other type of AddressEntity?

Reply:
Yeah as the 'AddressEntity' already exists just a way to separate them. Going to change the table back to just 'AddressEntity' once we switched to use this new relationship